dl/src/main/scala/com/intel/analytics/bigdl/optim/LocalOptimizer.scala

```diff
@@ -217,8 +217,9 @@ class LocalOptimizer[T: ClassTag] private[optim](
             val input = batch.data.narrow(1, offset + 1, length)
             val target = batch.labels.narrow(1, offset + 1, length)
             val output = workingModels(b).forward(input)
-            vMethods.map(validation => {
-              validation(output, target, workingCriterion(b))
+            val validatMethods = vMethods.clone()
```

Comment:
don't clone each time. use a buffer of vMehtods instead.
vMethods: Array[vMethod]
vMethods.length == parallelism
replace vMethods.clone() with vMethods(b)